dev-docs/source/ReleaseChecklist.rst

```diff
@@ -110,15 +110,22 @@ with the Release Manager.
 Monday, 3 weeks
 ---------------
 
-*  Ensure that Manual testing begins. The instructions for Manual testing are found
-   `here <https://www.mantidproject.org/Unscripted_Manual_Testing>`__.
+*  Ensure that Manual testing begins. An overview of what Manual testing is can be found
+   `here <https://www.mantidproject.org/Unscripted_Manual_Testing>`__. Generate the
```

Comment:
This is quite an old page. Do we want to link to the Testing index page instead? https://developer.mantidproject.org/Testing/index.html